addon/mixins/legend.js

```diff
@@ -88,6 +88,16 @@ export default Ember.Mixin.create({
     return this.get('numLegendRows') * this.get('legendItemHeight');
   }),
 
+  // Bottom margin is equal to the total amount of space the legend needs,
+  _marginBottom: Ember.computed('legendHeight', 'hasLegend', 'marginTop', function() {
+    // If the legend is enabled then we need some extra breathing room
+    return this.get('hasLegend') ? this.get('legendHeight') : this.get('marginBottom');
+  }),
+
+  marginBottom: Ember.computed('_marginBottom', 'minimumTopBottomMargin', function() {
+    return Math.max(this.get('_marginBottom'), this.get('minimumTopBottomMargin'));
+  }),
+
```

Comment:
In time-series-chart.js, we set the property with this name: `this.set('marginBottom', 20)`. What happens in such a scenario?